Changelog

```diff
@@ -1,9 +1,32 @@
 Changelog for Percona Toolkit
 
- * Improvement PT-1974: Support fingerprinting for --print in pt-kill (Thanks Iwo Panowicz)
- * Fixed bug   PT-1966: Fixed test (Thanks @yoku0825)
+v3.3.2 WIP release date not set yet
```

Comment:
Doesn't a release date need to be set before this is mergeable?

Reply:
No. The build script will set the release date